atomic_reactor/plugins/pre_pull_base_image.py

```diff
@@ -65,15 +65,22 @@ def run(self):
         Pull parent images and retag them uniquely for this build.
         """
         build_json = get_build_json()
-        base_image = self._resolve_base_image(build_json)
-        base_image_with_registry = self._ensure_image_registry(base_image)
+        base_image_str = str(self.workflow.builder.base_image)
+        for nonce, parent in enumerate(sorted(self.workflow.builder.parent_images.keys())):
```

Comment:
Might `nonce` contain ":" here? If so, won't using that value as an image tag fail?